src/EditorFeatures/VisualBasicTest/Formatting/Indentation/SmartIndenterTests.vb

```diff
@@ -2904,6 +2904,8 @@ End Class
                             Return p
                         End Function)
 
+                WpfTestCase.RequireWpfFact("Test helper creates mocks of ITextView")
```

Comment:
Well, the mock itself doesn't require it, but the code is assuming the thing behaves as a WPF editor.